crates/pixi_command_dispatcher/src/build/source_metadata_cache.rs

```diff
@@ -106,7 +109,7 @@ impl SourceMetadataCache {
         })?;
 
         // Try to acquire a lock on the cache file.
-        let cache_file_path = cache_dir.join(input.hash_key()).with_extension("json");
+        let cache_file_path = cache_dir.join("metadata.json");
```

Comment:
why it is `metadata.json`? I can't find a place where this file is created/written